content/docs/fairing/azure/az-kubeflow-notebook.md

```diff
@@ -0,0 +1,186 @@
+az aks show -g <resource-group-name> -n <name>
+```
+
+The preceding command has output like:
+
+```
+  "servicePrincipalProfile": {
+    "clientId": "<id>"
+  },
+```
+
+The role you grant must have at least read and write permissions to ACR and
+Storage.
+
+To learn more about how to grant the Service Principal access, follow the
+[Azure role-based authentication documentation][az-roles].
+
+## Setting up credentials as Kubernetes secrets
+
+Before running the notebook, you must first configure credentials so that the
+Python code running within the cluster can access the Azure resources required
+to train and deploy a model.
+
+Run the following commands to set up your credentials as a Kubernetes secret.
+
+1. Set environment variables to use in the following commands.
+
+   ```bash
+   export AZ_CLIENT_ID=<service-principal-client-id>
+   export AZ_CLIENT_SECRET=<service-principal-client-secret>
+   export AZ_TENANT_ID=<tenant-id>
+   export AZ_SUBSCRIPTION_ID=<subscription-id>
+   export TARGET_NAMESPACE=<target-namespace e.g. kubeflow-anonymous>
+   export ACR_NAME=<acr-name>
+   ```
+
+   * **AZ_CLIENT_ID:** The Service Principal client ID. You can get the
+     `client_id` property from ~/.azure/aksServicePrincipal.json.
+   * **AZ_CLIENT_SECRET:** The Service Principal secret. You can get the
+     `client_secret` property from ~/.azure/aksServicePrincipal.json.
+   * **AZ_TENANT_ID:** The Azure Tenant ID of your account. You can get the
+     Tenant ID from the `tenantId` field in the output of `az account show`.
+   * **AZ_SUBSCRIPTION:** The Azure Subscription ID of your account. You can
+     get the Subscription ID from the `id` field in the output of `az account
+     show`.
+   * **TARGET_NAMESPACE:** Specify the namespace that your Notebook Server is
+     in. For example, this guide recommends using `kubeflow-anonymous`.
+   * **ACR_NAME:** The name of an ACR that the Service Principal can access.
+
+1. Use the following command to create a secret that Kubeflow can use to access
+   Azure APIs.
+
+   ```bash
+   kubectl create secret generic -n ${TARGET_NAMESPACE} azcreds \
+   --from-literal=AZ_CLIENT_ID=${AZ_CLIENT_ID} \
+   --from-literal=AZ_CLIENT_SECRET=${AZ_CLIENT_SECRET} \
+   --from-literal=AZ_TENANT_ID=${AZ_TENANT_ID} \
+   --from-literal=AZ_SUBSCRIPTION_ID=${AZ_SUBSCRIPTION_ID}
+   ```
+
+1. Use the following command to create a secret that Kubeflow can use to access
+   ACR.
+
+   ```bash
+   kubectl create secret docker-registry -n ${TARGET_NAMESPACE} acrcreds \
+   --docker-server=${ACR_NAME}.azurecr.io \
+   --docker-username=${AZ_CLIENT_ID} \
+   --docker-password=${AZ_CLIENT_SECRET}
+   kubectl patch serviceaccount default-editor -n ${TARGET_NAMESPACE} \
+   -p "{\"imagePullSecrets\": [{\"name\": \"acrcreds\"}]}"
+   ```
+
+## Creating a Notebook Server in Kubeflow
+
+To create a Notebook Server, use your Web browser to access the Kubeflow
+Central Dashboard and select the `Notebook Servers` panel from the menu.
```

Comment:
When referring to UI text, use bold highlighting rather than code formatting:

**Notebook Servers**